blocks/results-list-block/features/results-list/default.jsx

```diff
@@ -82,6 +84,14 @@ ResultsList.propTypes = {
 			group: "Configure Content",
 			label: "Display Content Info",
 		}),
+		overline: PropTypes.string.tag({
```

Comment:
We shouldn't need to add custom fields here - we are not adding functionality

Reply:
removed overline and overlineurl customfields